submissions/Eugene-CG/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,81 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Eugene-CG/a-tiny-JS-world
+   Web app: https://eugene-cg.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const cat = {
+  species: "cat",
+  name: "Eugene",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "Hey Fellas!",
+};
+const dog = {
+  species: "dog",
+  name: "Toby",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "woof-woof!",
+};
+const man = {
+  species: "man",
+  name: "John",
+  gender: "male",
+  legs: 2,
+  hands: 2,
+  saying: "I am a man",
+};
+const woman = {
+  species: "dog",
+  name: "Yennefer",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: "Nice to meet you",
+};
+const catWoman = {
+  species: "Dream",
+  name: "Noone",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: cat.saying,
+};
+const arr = [cat, dog, man, woman, catWoman];
```

Comment:
Avoid using data types in the name of the variable. It is a clear indicator that your variable name isn't as obvious as it should be. Let's take this one for example. `arr` of what? If you use this variable outside this code somewhere, would you be able to understand what data is inside? Let's replace it with something like `inhabitants`. This gives you and understanding of what data is there and the fact that this is plural suggests that this is an array as well.